src/endToEnd.test.ts

```diff
@@ -3,19 +3,11 @@ import * as fs from 'fs'
 import * as path from 'path'
 import { app } from './server'
 import election from '../election.json'
-import { reset } from './store'
-import {
-  ballotImagesPath,
-  scannedBallotImagesPath,
-  sampleBallotImagesPath,
-} from './scanner'
+import { doZero } from './scanner'
+import { ballotImagesPath, sampleBallotImagesPath } from './scanner'
```

Comment:
Merge lines 6 and 7 as the methods come from the same file.

Reply:
👍